api/src/test/java/org/openmrs/api/PatientServiceTest.java

```diff
@@ -2751,6 +2751,14 @@ public void unvoidPatient_shouldUnvoidPerson() throws Exception {
 		Assert.assertFalse(patient.getPersonVoided());
 	}
 	
+	@Test
+	public void unvoidPatient_shouldReturnNullWhenPatientIsNull() throws Exception {
+		
+		PatientService patientService = Context.getPatientService();
+		Patient unVoidedPatient = patientService.unvoidPatient(null);
+		Assert.assertNull(unVoidedPatient);
```

Comment:
you can rewrite the test as (staticly importing assertNull)

```java
public void unvoidPatient_shouldReturnNullWhenPatientIsNull() throws Exception {

  assertNull(patientService.unvoidPatient(null);
}
```

Reply:
@teleivo I changed the code according to your comments. Feel free to review the code. Thanks